packages/eslint-plugin-pf-codemods/src/rules/helpers/renameSinglePropOnNode.ts

```diff
@@ -0,0 +1,94 @@
+import { JSXAttribute, JSXIdentifier, JSXOpeningElement } from 'estree-jsx';
+import { RenameConfig } from './renamePropsOnNode';
```

Comment:
Rather than renameSingle importing something from renamePropOnNode, and in turn renamePropOnNode having to import from renameSingle, could we define the RenameConfig in this file then just import both into renamePropOnNode?